.github/workflows/colcon_compile.yaml

```diff
@@ -0,0 +1,39 @@
+name: ros_colcon_CI
+on: [push, pull_request]
+
+env:
+  PACKAGES_TO_TEST:
```

Comment:
how did u get this list? is this list complete? i dont think it is?

![image](https://user-images.githubusercontent.com/52610118/209238655-791750dc-1d4e-4f59-9ff1-3a5cb7df2a11.png)

Reply:
I just ran rospack and put down a bunch of the "movo" packages. I can try the other ones and I also have to look into why movo_demos is not building with this